iina/MPVController.swift

```diff
@@ -554,21 +541,6 @@ class MPVController: NSObject {
     mpv_set_property(mpv, name, MPV_FORMAT_DOUBLE, &data)
   }
 
-  func setFlagAsync(_ name: String, _ flag: Bool) {
-    var data: Int = flag ? 1 : 0
-    mpv_set_property_async(mpv, 0, name, MPV_FORMAT_FLAG, &data)
-  }
-
-  func setIntAsync(_ name: String, _ value: Int) {
-    var data = Int64(value)
-    mpv_set_property_async(mpv, 0, name, MPV_FORMAT_INT64, &data)
-  }
-
-  func setDoubleAsync(_ name: String, _ value: Double) {
-    var data = value
-    mpv_set_property_async(mpv, 0, name, MPV_FORMAT_DOUBLE, &data)
-  }
-
```

Comment:
These functions were added intentionally, in the future we might want to use them. Please convert this back.